ansible/group_vars/gelato-ha.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Dummy variable to avoid error because ansible does not recognize the
+# file as a good configuration file when no variable in it.
+dummy:
+
+
+###########
+# GENERAL #
+###########
+
+#Polulate these IP , Port once the services are installed on K8s.
+gelato_ha_api_ip: API_IP
+gelato_ha_api_port: API_PORT
+
+gelato_ha_s3_ip: S3_IP
+gelato_ha_s3_port: S3_PORT
+
+# These fields are NOT suggested to be modifie
+gelato_work_dir: /opt/opensds-gelato-linux-amd64
+
+# The installation type of gelato is container, both REPOSITORY and RELEASE will
+# use gelato image to deploy.
+gelato_image_tag: "{{ gelato_release_version }}"
```

Comment:
Question: The HA release version and  the standalone release version are going to remain the same?
If yes, then this is ok.

Reply:
Yes, will be served out of same code base.